In file included from Unified_cpp_accessible_html0.cpp:29:
In file included from accessible/html/HTMLImageMapAccessible.cpp:14:
In file included from layout/generic/nsImageFrame.h:17:
In file included from ../.build/dist/include/nsDisplayList.h:50:
In file included from ../.build/dist/include/nsCSSRenderingBorders.h:20:
In file included from ../.build/dist/include/gfxUtils.h:22:
In file included from ../.build/dist/include/mozilla/webrender/WebRenderTypes.h:11:
In file included from ../.build/dist/include/mozilla/webrender/webrender_ffi.h:108:
../.build/dist/include/mozilla/webrender/webrender_ffi_generated.h:gmake[4]: Leaving directory '../.build/accessible/aom'
24:33: error: redefinition of 'ROOT_CLIP_CHAIN'
constexpr static const uint64_t ROOT_CLIP_CHAIN = ~0;
                                ^
../.build/dist/include/mozilla/webrender/webrender_ffi.h:80:16: note: previous definition is here
const uint64_t ROOT_CLIP_CHAIN = ~0;
               ^

--- gfx/webrender_bindings/webrender_ffi.h.orig	2022-05-23 17:12:36 UTC
+++ gfx/webrender_bindings/webrender_ffi.h
@@ -77,8 +77,6 @@ using WrPipelineIdEpochs = nsTArray<WrPipelineIdAndEpo
 struct WrPipelineIdAndEpoch;
 using WrPipelineIdEpochs = nsTArray<WrPipelineIdAndEpoch>;
 
-const uint64_t ROOT_CLIP_CHAIN = ~0;
-
 }  // namespace wr
 }  // namespace mozilla
 
